deployments/kubehound/docker-compose.dev.ingestor.yaml

```diff
@@ -9,6 +9,10 @@ services:
       - "127.0.0.1:9000:9000"
     networks:
       - kubenet
+    env_file:
+      - kubehound.env
+    labels:
+      com.datadoghq.ad.logs: '[{"app": "grpc", "service": "kubehound"}]'
```

Comment:
seems weird to have `app:grpc` tbh

Reply:
I just put the name of container for the `app`. Same as all the others labels.
This is only for local dev anyway.